Description: add latest change date to manpage
 $4 is not initialized / used, add a static date instead.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Bug-Debian: http://bugs.debian.org/674052
Last-Update: 2013-11-04

---

--- patch-2.7.1.orig/patch.man
+++ patch-2.7.1/patch.man
@@ -7,7 +7,7 @@
 .if t .sp .3
 .if n .sp
 ..
-.TH PATCH 1 \*(Dt GNU
+.TH PATCH 1 2012-09-28 GNU
 .ta 3n
 .SH NAME
 patch \- apply a diff file to an original
